### PR TITLE
Advisor info

### PR DESCRIPTION
**Pull request by karelhala**
selection_012

### CODE REVIEW

**Comment by AllenBW** [comment]
essential https://github.com/RedHatInsights/insights-advisor-frontend/pull/100 cept here, upon merging this, we're gonna have to revert https://github.com/RedHatInsights/insights-advisor-frontend/pull/100 in advisor frontend

File: src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                        <SplitItem><Battery label='Risk Of Change' severity={ rule.resolution_risk } /></SplitItem>
+                    </Split>
+                </CardHeader>
+                { expanded && (<CardBody>
+                    <Grid gutter='md' sm={ 12 }>
+                        <GridItem>
+                            <Card className='pf-t-light  pf-m-opaque-100'>
+                                <CardHeader> <ThumbsUpIcon /> Detected Issues</CardHeader>
+                                <CardBody>
+                                    <div dangerouslySetInnerHTML={ { __html: rule.reason_html } } />
```
Comment (dkuc) [inline src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js]: FYI, we plan to remove these _html properties from the rule.  in this case you would just use .reason

File: src/SmartComponents/Inventory/applications/advisor/Advisor.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+    expandAll(expanded) {
+        this.setState({ expanded: !expanded });
+    }
+
+    buildRuleCards = () => {
+        const { inventoryReport, expanded } = this.state;
+        return (
+            <Fragment>
+                <div className="pf-u-display-flex pf-u-flex-direction-row-reverse">
```
Comment (AllenBW) [inline src/SmartComponents/Inventory/applications/advisor/Advisor.js]: This doesn't appear to be showing up correctly, should be in upper right of the list

File: src/SmartComponents/Inventory/applications/advisor/Advisor.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                        </Card>
+                ) }
+                { inventoryReportFetchStatus === 'failed' && (
+                    <Card className="ins-empty-rule-cards">
+                        <CardHeader>
+                            <FrownOpenIcon size='lg' />
+                        </CardHeader>
+                        <CardBody>
+                            There was an error while trying to fetch rules list for this entity. Please contact your
+                            administrator and show him this screen.
```
Comment (AllenBW) [inline src/SmartComponents/Inventory/applications/advisor/Advisor.js]: Gender bias alarms going off, unless this is a uxd sanctioned message would recommend considering the following rewording: 

" There was an error fetching rules list for this Entity. Please show your administrator this screen."

File: src/SmartComponents/Inventory/applications/advisor/Advisor.js (hunk 2 above)
Comment (karelhala) [reply src/SmartComponents/Inventory/applications/advisor/Advisor.js]: You ought to `import '@red-hat-insights/insights-frontend-components/components/Advisor.css'` as well.

Comment (karelhala) [reply src/SmartComponents/Inventory/applications/advisor/Advisor.js]: Oh I see, mmhh the problem here is that we are currently using old version of pf-next in FE components. So you have to import '@patternfly/patternfly-next/utilities/Flex/flex.css` on your own so this text is on right

Comment (AllenBW) [reply src/SmartComponents/Inventory/applications/advisor/Advisor.js]: 👍 🤗

**Comment by karelhala** [comment]
@AllenBW updated, please retest and re-review.

**Pull request merged**